README.md

```diff
@@ -11,13 +11,13 @@
 
 # Webpack CLI
 
-This project intends to be the main cli package of webpack. Here we will encapsulate all the features and code related to the command line. From sending options to the compiler, to initialize and migrate from version to version. To avoid breaking changes and making people unhappy, we are keeping the old features. While we keep the old features, we've added some other, `init`and `migrate`. 
+Webpack CLI encapsulates all code related to CLI handling. It captures options and sends them to webpack compiler. You can also find functionality for initializing a project and migrating between versions. For the time being, it is backwards-compatible with the CLI included in webpack itself.
```

Comment:
`Webpack CLI encapsulates all code related to CLI handling` I know you've already changed this once. I wonder if there's a terse one liner that can explain what CLI handling gives end-users in this context. 

`Webpack CLI interacts with your build - it offers everything webpack's current CLI does, in addition to project scaffolding and migration`. or something.